File: 1-Two-Sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> res(2);
        for(int i=0;i<nums.size();i++)
        {
            for(int j=i;j<nums.size();j++){
                if(i!=j)
                {
                    if(nums[i]+nums[j]==target)
                    {
                        res[0]=i;
                        res[1]=j;
                        return res;
                    }
                }
            }
        }
       return res;  
    }
};

File: 1-Two-Sum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        int n=nums.length;
        for(int i=1;i<n;i++)
        {
           for(int j=i;j<n;j++)
           {
            int l=j-i;
             if(nums[l]+nums[j]==target)
             {
                return new int[] {l,j};
             }
           }
        }
        return new int[0];
                
    }
}